Revit.Async.Shared/RevitTask.cs

```diff
@@ -70,15 +49,30 @@ public void Dispose()
         {
             _scopedRegisteredExternalEvents.Clear();
             _scopedRegisteredExternalEvents = null;
+#if DEBUG
+            Writer?.Close();
```

Comment:
@KennanChan I was unable to build in Debug without commenting out these two lines. I don't see where the `Writer` property is declared, but maybe I'm missing something?